specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2023-08-01-preview/IncidentCreationRules.json

```diff
@@ -0,0 +1,640 @@
+            "description": "The severity of the alert."
+          },
+          {
+            "value": "AlertType",
+            "description": "The type of the alert."
+          }
+        ]
+      }
+    },
+    "IncidentCreationRulePropertyGroupingSupportedProperty": {
+      "description": "The property to evaluate by which we will group alerts.",
+      "enum": [
+        "Entities",
+        "CustomDetails",
+        "AlertProperties"
+      ],
+      "type": "string",
+      "example": "Entities",
+      "x-ms-enum": {
+        "name": "IncidentCreationRulePropertyGroupingSupportedProperty",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Entities",
+            "description": "Group alerts by identical entities."
+          },
+          {
+            "value": "CustomDetails",
+            "description": "Group alerts by custom details."
+          },
+          {
+            "value": "AlertProperties",
+            "description": "Group alerts by alert properties."
+          }
+        ]
+      }
+    },
+    "IncidentCreationRulePropertyConditionProperties": {
+      "type": "object",
+      "properties": {
+        "propertyName": {
+          "$ref": "#/definitions/IncidentCreationRulePropertyConditionSupportedProperty"
+        },
+        "operator": {
+          "$ref": "#/definitions/IncidentCreationRulePropertyConditionSupportedOperator"
+        },
+        "propertyValues": {
+          "maxItems": 50,
+          "minItems": 1,
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "IncidentCreationRulesList": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IncidentCreationRule"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string"
+        }
+      }
+    },
+    "PropertyBasedGroupingProperties": {
+      "description": "Describes a incident creation rule grouping configuration that evaluates properties values",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/IncidentCreationRuleGroupingConfiguration"
+        }
+      ],
+      "properties": {
+        "groupingProperties": {
+          "maxItems": 5,
+          "type": "object",
+          "$ref": "#/definitions/IncidentCreationRulePropertyBasedGroupingProperties"
+        }
+      },
+      "x-ms-discriminator-value": "PropertyBased",
+      "x-ms-client-flatten": true
+    },
+    "IncidentCreationPropertyConditionProperties": {
+      "description": "Describes a incident creation rule condition that evaluates a property's value",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/IncidentCreationRuleCondition"
+        }
+      ],
+      "properties": {
+        "conditionProperties": {
```

Comment:
>conditionProperties [](http://example.com/codeflow?start=9&length=19)

This looks like a single condition to me. Maybe condition is a better property name?